### PR TITLE
Add alarm subscription readme

### PR DESCRIPTION
**Pull request by Jennifer-chen-rh**
Readme update for alarm subscription server

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -238,3 +238,43 @@ Notes:
 * This API is not defined by O2ims Interface Specification.
 * The server supports the `alarmProbableCauses` endpoint for exposing a custom list of probable causes.
 * The list is available in [data folder](internal/files/alarms/probable_causes.json). Can be customized and maintained as required.
+
+#### Alarm Subscription server
+
+The alarm subscription server needs to connect to the non-kubernetes API of the
```
Comment (jhernand) [inline README.md:244]: I think isn't completely accurate: the alarm subscription server doesn't need to connect to the non-kubernetes API of the global hub. It only connects to the Kubernetes API to read and write the configmaps it uses to store subscriptions. I think you can remove this first sentence.

File: README.md [hunk 2]
```diff
@@ -238,3 +238,43 @@ Notes:
...
+Start the deployment manager server with a command like this:
+
+```
+$./oran-o2ims start alarm-subscription-server \
+--log-file="servers.log" \
+--log-level="debug" \
+--log-field="server=alarm-subscription" \
+--log-field="pid=%p" \
+--api-listener-address="127.0.0.1:8006" \
+--metrics-listener-address="127.0.0.1:8008" \
```
Comment (danielerez) [inline README.md:256]: Not related to this PR, but I guess worth adding this also to the other examples in this readme later.

File: README.md [hunk 3]
```diff
@@ -238,3 +238,43 @@ Notes:
...
+--cloud-id="123" 
+```
+
+Note that by default all the servers listen on `localhost:8000`, so there will
+be conflicts if you try to run multiple servers in the same machine. The
+`--api-listener-address` and `metrics-listener-address` options are used to select a port number that isn't in use.
```
Comment (danielerez) [inline README.md:262]: nit: to keep convention, consider adding the '--' prefix to both, i.e. `--metrics-listener-address`.

File: README.md [hunk 4]
```diff
@@ -238,3 +238,43 @@ Notes:
...
+
+The `cloud-id` is any string that you want to use as identifier of the O-Cloud instance.
+
+For more information about other command line flags use the `--help` command:
+
+```
+$ ./oran-o2ims start alarm-subscription-server --help
+```
+Note
```
Comment (danielerez) [inline README.md:271]: This line seems redundant?

File: README.md (hunk 3 above)
Comment (Jennifer-chen-rh) [reply README.md]: Thanks for the comments and Addressed. And add the option to other examples as well.

File: README.md (hunk 2 above)
Comment (Jennifer-chen-rh) [reply README.md:259]: Add to other examples as well.

**Comment by Jennifer-chen-rh** [comment]
@jhernand @danielerez @irinamihai Any more comments? Can we merge it? thanks

**Pull request merged**